src/main/java/org/folio/circulation/domain/representations/CheckOutByBarcodeRequest.java

```diff
@@ -55,4 +61,16 @@ public Loan toLoan() {
 
     return Loan.from(loanJson);
   }
+
+  public boolean isItemLimitBlockOverriding() {
+    return this.overrideBlocks != null && this.overrideBlocks.getItemLimitBlock() != null;
+  }
+
+  public boolean isPatronBlockOverriding() {
+    return this.overrideBlocks != null && this.overrideBlocks.getPatronBlock() != null;
+  }
+
+  public boolean isItemNotLoanableBlock() {
+    return this.overrideBlocks != null && this.overrideBlocks.getItemNotLoanableBlock() != null;
+  }
```

Comment:
Move this to OverrideBlocks, then we can remove "Override" part from the name - e.g. containsPatronBclock etc.